client/flutter/ios/fastlane/Fastfile

```diff
@@ -0,0 +1,36 @@
+# This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+default_platform(:ios)
+
+platform :ios do
+  desc "Push a new beta build to TestFlight"
+  lane :beta do
+    build_app(workspace: "Runner.xcworkspace", scheme: "Runner")
+    upload_to_testflight
+  end
+
+  desc "Promote the beta build to App Store"
+  lane :submit_review do
+    deliver(
+      submit_for_review: true,
+      automatic_release: true,
```

Comment:
We'd like to do automatic upload but manual confirmation of release. Will this automatically release a new version based once approved? If so, then please change this to `false`. Given lots of localized binaries, we'd probably need a separate command that did the actual release. I'm not sure how many of the binaries will go through Apple review.

Reply:
Yes it would automatically release version once approved. Change done.